x-pack/plugins/cloud/public/services/index.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, createContext, useContext } from 'react';
+
+interface WithoutChat {
+  enabled: false;
+}
+
+interface WithChat {
+  enabled: true;
+  chatURL: string;
+  identityJWT: string;
+  userID: string;
+  userEmail: string;
+}
+
+export type ChatService = WithChat | WithoutChat;
```

Comment:
NIT/optional: misleading type name, this holds configuration values, I don't think it should be named `*Service`.